src/FilenameParsing/AbstractFileIDHelper.php

```diff
@@ -2,12 +2,28 @@
 
 namespace SilverStripe\Assets\FilenameParsing;
 
+use SilverStripe\Core\Convert;
 use SilverStripe\Core\Injector\Injectable;
 
 abstract class AbstractFileIDHelper implements FileIDHelper
 {
     use Injectable;
 
+    /**
+     * A variant type for encoding a variant filename with a different extension than the original.
+     */
+    public const EXTENSION_REWRITE_VARIANT = 'extRewrite';
```

Comment:
```suggestion
    public const EXTENSION_REWRITE_VARIANT = 'ExtRewrite';
```
All the other variants types have initial caps

Reply:
That's just a fluke because the method names start with a capital letter - but I'll change this to avoid PR ping pong

Reply:
Done